### ISSUE TITLE
"*" does nothing on Ubuntu 24.04

### ISSUE THREAD

**Issue by El-Barto89** [linuxmint/timeshift#313]
**Describe the bug**
If I have:
+/home/user/.conf/**
+/home/user/1_utility/**
/home/user/**
/root/**
in this exact order, I dont get only the .config and 1_utility folder created and synced (They should be empty, of course)

**To Reproduce**
1. setting up filter
2. click ok 
3. run timeshift

**Expected behavior**
Creation of the folder only

**System:**
 - Ubuntu 24.04
 - Desktop GNOME 46
 - Application Version [v 24.01.1]

**Comment by teejee2008** [comment]
Read the Rsync docs to understand how the filters work. Only the first 2 filters in your list are getting used since they match everything. After Rsync finds a match it will ignore the remaining filters in the list.

**Comment by El-Barto89** [comment]
Yes, I want only the first 2 filter to be used, but with /** I dont get only empy ".config" and "1_utility" folders, they are ignored. I do want only .config and 1_Utility folder, but both empty.
